sdk-actors/src/main/java/io/dapr/actors/runtime/ActorRuntime.java

```diff
@@ -112,29 +112,37 @@ public static ActorRuntime getInstance() {
    * Registers an actor with the runtime.
    *
    * @param clazz            The type of actor.
-   * @param objectSerializer Serializer for Actor's state and transient objects.
+   * @param objectSerializer Serializer for Actor's transient objects.
+   * @param stateSerializer  Serializer for Actor's state objects.
    * @param <T>              Actor class type.
    */
-  public <T extends AbstractActor> void registerActor(Class<T> clazz, DaprObjectSerializer objectSerializer) {
-    registerActor(clazz, null, objectSerializer);
+  public <T extends AbstractActor> void registerActor(
+    Class<T> clazz, DaprObjectSerializer objectSerializer, DaprObjectSerializer stateSerializer) {
```

Comment:
I know why we let users to pass these serializers, but as a developer, I do not like to set serializer explicitly all the time :) this is my personal opinion.

Reply:
We had a strong feedback on this. On the other hand, we can quickly add overloaded methods hiding this based on user feedback.